vhdbuilder/packer/imagecustomizer/azlosguard/azlosguard.yml

```diff
@@ -0,0 +1,316 @@
+os:
+  additionalFiles:
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/azlosguard/10-users-ssh-enable.cfg
+      destination: /etc/cloud/cloud.cfg.d/10-users-ssh-enable.cfg
+      permissions: 644
+    - source: /AgentBaker/parts/linux/cloud-init/artifacts/azlosguard/azurelinux-cloud-native.repo
```

Comment:
this whole config is basically a packer template + `packer_source.sh` replacement right?

Reply:
Yeah, the bulk of this config is for copying files from AgentBaker into the VHD's filesystem with the right permissions set, along with enabling (and disabling) some services.